dbt-cta/usvote_foundation/models/0_ctes/offices_ab4.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(
+    cluster_by = "_airbyte_emitted_at",
+    partition_by = {"field": "_airbyte_emitted_at", "data_type": "timestamp", "granularity": "day"},
+    unique_key = '_airbyte_offices_hashid'
+) }}
+
+-- SQL model to dedupe on the hashid field calculated in *_ab3
+-- (this step is included because the data source seems to include duplicate rows)
+-- depends_on: {{ ref('offices_ab3') }}
+
+SELECT * FROM 
+(
+SELECT 
+    *,
+    ROW_NUMBER() OVER (PARTITION BY _airbyte_offices_hashid ORDER BY _airbyte_ab_id desc) as rownum 
```

Comment:
would it be better to order by `_airbyte_emitted_at`? that way were always grabbing the latest record

Reply:
oh, yeah good call! since these are dupes in the actual raw data I think the difference between `_airbyte_emitted_at` values would be on the order of seconds or even milliseconds, but as long as we're ordering by something, it's slightly less arbitrary lol. I'll push that change!